week-2/D-predicates/exercise.js

```diff
@@ -7,12 +7,14 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+  return number === 5 && number < 0
```

Comment:
This function always returns `false`, because a number can't both be 5 _and_ be less than 0 - can you give this one another go? :)